### PR TITLE
LPD-31933 - Address ERC gaps in Headless APIs for shipmentItems

### PR DESCRIPTION
**Pull request by brbalazs**
https://liferay.atlassian.net/browse/LPD-31933

Resent from https://github.com/brianchandotcom/liferay-portal/pull/153246

### CODE REVIEW

**Comment by brbalazs** [comment]
ci:test:sf

**Comment by brbalazs** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/util/v1_0/ShipmentItemUtil.java [hunk 1]
```diff
@@ -24,6 +28,8 @@ public class ShipmentItemUtil {
 
 	public static CommerceShipmentItem addOrUpdateShipmentItem(
 			String externalReferenceCode, CommerceShipment commerceShipment,
+			CommerceInventoryWarehouseService commerceInventoryWarehouseService,
```
Comment (smottal) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/util/v1_0/ShipmentItemUtil.java:31]: these are not sorted

**Review by smottal — CHANGES_REQUESTED** [review]

**Comment by brbalazs** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**